docs/source/user/api_change.rst

```diff
@@ -10,19 +10,33 @@ The line number corresponds to the resulting line number after all changes are i
 Thus, be sure to implement each in order so that subsequent line numbers are correct.
 
 
-OpenFAST v2.6.0 to OpenFAST `dev`
+OpenFAST v3.0.0 to OpenFAST `dev`
 ---------------------------------
 No changes.
 
 
 
+OpenFAST v2.6.0 to OpenFAST v3.0.0
+----------------------------------
+No changes.
+
+
+
```

Comment:
The ServoDyn changes below appear to be misplaced.  These are added in v3.0.0, not v2.6.0.